Tasks/Common/webdeployment-common-v2/npmdomutility.ts

```diff
@@ -0,0 +1,98 @@
+var varUtility = require("./variableutility.js");
+var DOMParser = require('xmldom').DOMParser;
+
+export class NpmDomUtility  {
+
+    private xmlDomLookUpTable = {};
+    private headerContent;
+    private xmlDom;
+
+    public constructor(xmlContent) {
+        this.xmlDomLookUpTable = {};
+        this.headerContent = null;
+        this.xmlDom = new DOMParser().parseFromString(xmlContent,"text/xml");
+        this.readHeader(xmlContent);
+        this.buildLookUpTable(this.xmlDom);
+    }
+
+    public getXmlDom() {
+        return this.xmlDom;
+    }
+
+    private readHeader(xmlContent) {
+        var index = xmlContent.indexOf('\n');
```

Comment:
Change all "var" declarations to "let" wherever possible. A variable declared with var is defined throughout the program as compared to let.